src/extension/background-script/state.ts

```diff
@@ -214,12 +214,14 @@ const state = create<State>((set, get) => ({
 
     // https://stackoverflow.com/a/54317362/1667461
     const allTabIds = Array.from(allTabs, (tab) => tab.id).filter(
-      (i): i is number => {
-        return typeof i === "number";
+      (id, index): id is number => {
+        // Safari: allTabs consist of Start Pages too
+        return typeof id === "number" && allTabs[index].title !== "";
```

Comment:
Line 213 queries all tabs with the Title "Alby", however here we filter out tabs with empty titles? They shouldn't be included in this list, right?

Reply:
Yeah that's something done for Safari, the start page tabs also get queried (which is weird as they have `title:""`) so here we are removing those again so we don't close the window on closing Alby in the start page (given there are no other tabs opened in Safari with proper url)

Safari is weird.